components/public-api/gitpod/experimental/v1/workspaces.proto

```diff
@@ -111,6 +114,12 @@ message DeleteWorkspaceRequest {
 
 message DeleteWorkspaceResponse {}
 
+message ListWorkspaceClassesRequest {}
+
+message ListWorkspaceClassesResponse {
+    repeated WorkspaceClass result = 1;
```

Comment:
Mostly for my own understanding, do we typically try to put the main response in a `result` property? I've noticed some do and some don't.

Reply:
@selfcontained I have done result for the methods I've implemented so far copying how ListWorkspaces does it. I don't think we have any guidelines / conventions for that yet.

Reply:
Yes, we do that for proto definitions.